fictadvisor-api/src/v2/api/controllers/UserController.ts

```diff
@@ -812,6 +812,34 @@ export class UserController {
   }
 
   @Access(PERMISSION.USERS_$USERID_TELEGRAM_LINK)
+  @ApiOkResponse({
+    type: OrdinaryStudentResponse,
+    description: 'Added user\'s telegram link',
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
```

Comment:
Add Telegram DTO Exceptions and InvalidTelegramCredentialsException with DuplicateTelegramIdException